spec/index.md

```diff
@@ -5064,7 +5065,7 @@ S with
     reserved_balance_limits[Canister_id] = New_reserved_balance_limit
     certified_data[Canister_id] = ""
     canister_history[Canister_id] = New_canister_history
-    canister_log_visibility[Canister_id] = Public
+    canister_log_visibility[Canister_id] = New_controllers
```

Comment:
The default to Public is set at multiple places - could you please update them all?

Reply:
Done. Also I was not sure if order is important, but just in case I switched to `Controllers | Public`.